lib/3scale/backend/async_redis/client.rb

```diff
@@ -19,7 +19,7 @@ def call(opts)
             uri = URI(opts[:url])
 
             credentials = [ uri.user || opts[:username], uri.password || opts[:password]]
-            db = uri.path[1..-1].to_i if uri.path
+            db = uri.path[1..-1] if uri.path
```

Comment:
could you add a test that in absence of a `db` part of URL, the protocol will not set that? I think it is worth it.

Reply:
Sure